sycl/test-e2e/BFloat16/bfloat16_vec.cpp

```diff
@@ -0,0 +1,159 @@
+//==------------ bfloat16_vec.cpp - test sycl::vec<bfloat16>----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: cpu || accelerator
```

Comment:
Why? If simply not yet supported there, add a comment stating that.

Reply:
added